ThankIt/ThankIt/Presentation/Main/MainView.swift

```diff
@@ -6,41 +6,107 @@
 //
 
 import SwiftUI
+import PopupView
 
 struct MainView: View {
+    @AppStorage("userNickname") private var userNickname: String = ""
+    @State var selectedFlavor: UserScope = .all
+    @State private var selectedThank: Thank? = nil
+    
+    private(set) var columns: [GridItem] = Array(repeating: .init(.flexible()), count: 2)
+
     let thanks: [Thank]
+
     var body: some View {
-        List {
-            ForEach(thanks) { thank in
-                PostItView(thank: thank)
+        NavigationStack {
+            ZStack {
+                ScrollView {
+                    // MARK: Picker
+                    HStack {
+                        Spacer()
+                        
+                        Picker("UserScope", selection: $selectedFlavor) {
+                            Text(UserScope.all.rawValue).tag(UserScope.all)
+                            Text(UserScope.me.rawValue).tag(UserScope.me)
+                        }
+                        .pickerStyle(.segmented)
+                        .frame(width: Metrics.pickerFrame)
+                        .padding()
+                    }
```

Comment:
```suggestion
                        Picker("UserScope", selection: $selectedFlavor) {
                            Text(UserScope.all.rawValue).tag(UserScope.all)
                            Text(UserScope.me.rawValue).tag(UserScope.me)
                        }
                        .pickerStyle(.segmented)
                        .frame(width: Metrics.pickerFrame)
                        .padding()
                        .frame(maxWidth: .infinity, alignment: .trailing)
```
이렇게 쓸 수도 있을 것 같아요! 
앞에서 프레임으로 피커의 크기를 잡아준 후, 외부에 padding을 포함한 뷰를 반환하고 있어서, 
해당 뷰를 그냥 infinity로 늘려주시고 정렬 잡아주시면 될 것 같습니다!

Reply:
저도 My의 코드를 리뷰하다가 ` .frame(maxWidth: .infinity, alignment: .trailing)`같이 정렬하는 방식을 발견해서 요긴하게 쓰고 있습니다... ㅎㅎ